app/services/usps_in_person_proofing/mock/proofer.rb

```diff
@@ -40,8 +40,12 @@ def request_facilities(_location, is_enhanced_ipp)
         end
       end
 
-      def request_proofing_results(_enrollment)
-        JSON.parse(Fixtures.request_passed_proofing_results_response)
+      def request_proofing_results(enrollment)
+        if enrollment.days_to_due_date.negative? && enrollment.enhanced_ipp?
```

Comment:
I added this to support manual testing.  Feedback is welcome!  Making the manual testing plan took me longer than anticipated.